src/cljc/slurper/text.cljc

```diff
@@ -110,21 +110,22 @@
           (when (some? i)
```

Comment:
Not sure why this changes are in PR, because they were in
https://github.com/JetBrains/andel/commit/257d13f5821528daf35cbf5f98278537b16f6d91